docs/content/overview/introduction.md

```diff
@@ -133,6 +133,7 @@ Hugo boasts the following features:
 ### Additional Features
 
   * Integrated [Disqus](https://disqus.com/) comment support
+  * [txtpen](https://txtpen.com/) highlights and in-text commenting support
```

Comment:
Not a knock on Txtpen by any means (very cool project, btw), but I don't think it's going to have the same draw as our internal templates for the most common use cases (i.e., GA and Disqus).